DiscordChatExporter.Cli/Commands/ExportAllCommand.cs

```diff
@@ -66,6 +66,22 @@ await foreach (var guild in Discord.GetUserGuildsAsync(cancellationToken))
                     if (!IncludeVoiceChannels && channel.Kind.IsVoice())
                         continue;
 
+                    // if --after is specified, skip channels with no new messages
+                    if (
+                        After != null
+                        && channel.LastMessageId != null
+                        && After > channel.LastMessageId
+                    )
+                        continue;
+
+                    // if --before is specified, skip channels created after the specified date
+                    if (Before != null && Before < channel.Id)
+                        continue;
```

Comment:
We do have these checks in `ChannelExporter`, don't we?

Reply:
You are right. I reverted checks in `ExportAllCommand` and `ExportGuildCommand` and added missing --before check to `ChannelExporter`